core/src/main/java/org/mule/runtime/core/internal/exception/OnErrorPropagateHandler.java

```diff
@@ -7,15 +7,21 @@
 package org.mule.runtime.core.internal.exception;
 
 import static reactor.core.publisher.Mono.just;
+
+import org.mule.runtime.api.component.location.ComponentLocation;
 import org.mule.runtime.core.api.event.CoreEvent;
 import org.mule.runtime.core.api.processor.Processor;
+import org.mule.runtime.core.api.transaction.Transaction;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.mule.runtime.core.api.transaction.Transaction'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7640&issues=AWirRZ1EotIXnmsUDiqP&open=AWirRZ1EotIXnmsUDiqP)

[ISSUE_KEY]: AWirRZ1EotIXnmsUDiqP